src/utils/weather.js

```diff
@@ -1,13 +1,46 @@
 import axios from 'axios';
 
-export const currentWeather = async () => {
-  const url = `http://api.openweathermap.org/data/2.5/weather?lat=30.267153&lon=-97.7430608&appid=${
+export const renderWeatherImg = img => `wi wi-owm-${img}`;
+
+const getServiceUrl = (service, latitude, longitude) =>
```

Comment:
you can just name this `serviceUrl`; adding `get` or `fetch` tends to make other devs think you are making a request here.

Reply:
done